activemq_xml/tests/conftest.py

```diff
@@ -19,7 +20,11 @@ def dd_environment():
     with TempDir() as tmp_dir:
         activemq_data_dir = os.path.join(tmp_dir, "data")
         fixture_archive = os.path.join(HERE, "fixtures", "apache-activemq-kahadb.tar.gz")
-        os.mkdir(activemq_data_dir)
+        try:
+            os.mkdir(activemq_data_dir)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                raise
```

Comment:
Let's check for existence instead like this https://github.com/DataDog/integrations-core/blob/7d6c9a3b7e374e646d7dd69b6da4ecf599192ed9/mcache/tests/conftest.py#L39-L40